prog/main.go

```diff
@@ -134,6 +136,52 @@ type appFlags struct {
 	consulInf       string
 }
 
+type containerLabelFilterFlagArray []string
```

Comment:
`s/containerLabelFilterFlagArray/containerLabelFilters/` and instead of using an array of strings use an array of the final parsed values (which you should parse in `Set()`. Something like

```go
type containerLabelFilter struct {
    title string
    label struct {
      key string
      value string
    }
}

type containerLabelFilters []containerLabelFilter
```

Then you can parse it in the `Set()` method and add a `toAPITopologyOption()` method (replacing `makeContainerFiltersFromFlags`)